polkadot/runtime/parachains/src/assigner_on_demand/mod.rs

```diff
@@ -368,10 +368,10 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// An order was placed at some spot price amount.
-		OnDemandOrderPlaced { para_id: ParaId, spot_price: BalanceOf<T> },
-		/// The value of the spot traffic multiplier changed.
-		SpotTrafficSet { traffic: FixedU128 },
+		/// An order was placed at some spot price amount by orderer ordered_by
+		OnDemandOrderPlaced { para_id: ParaId, spot_price: BalanceOf<T>, ordered_by: T::AccountId },
+		/// The value of the spot price has likely changed
+		SpotPriceSet { spot_price: BalanceOf<T> },
```

Comment:
This *may* be problematic as you're changing the formats of the existing events here. As events from the previous block are reset after the runtime upgrade but before running any hooks, [event storage must always be compatible](https://github.com/paritytech/polkadot-sdk/blob/2b1c606a338c80c5220c502c56a4b489f6d51488/substrate/frame/executive/src/lib.rs#L577-L579). So, technically, this part needs more care.

Buuuut, given that, IIUC,
1) `OnDemandOrderPlaced` was never deposited by the previous version of the code, and
2) `SpotPriceSet` internal representation is the same, as `BalanceOf<T>` coincides with `FixedU128` in all the current use cases,

it may be safe to leave it as is.

That said, some more research is needed on how safe it is.

CC @liamaharon @bkchr